pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -174,10 +197,44 @@ func (c *MizarArktosNetworkController) syncNetwork(eventKeyWithType KeyWithEvent
 }
 
 func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Network, eventKeyWithType KeyWithEventType) error {
+	//Find out the paths of default template to create vpc and subnet
+	vpc := network.Spec.VPCID
+	subnet := vpc + subnetSuffix
+
+	currentDir, err := os.Getwd()
+	if err != nil {
+		klog.Errorf("Mizar-Arktos-Network-controller: Get current directory (%s) in error (%v).", currentDir, err)
+		return err
+	}
+
+	if !strings.HasSuffix(currentDir, arktosName) {
+		klog.Errorf("Mizar-Arktos-Network-controller: Current directory (%s) is not in Arktos Home directory with error (%v).", currentDir, err)
+		return err
+	}
+
+	templateDir := currentDir + homeSubPath
+
+	vpcDefaultTemplatePath := templateDir + "default_mizar_network_vpc_template.json"
+	subnetDefaultTemplatePath := templateDir + "default_mizar_network_subnet_template.json"
```

Comment:
All the path generation only need to be initialized once. No need to happen in processNetworkCreation

Reply:
Thanks for this good finding!

Reply:
Will work on this last feedback.

Reply:
All the path generation is initialized once in cmd/kube-controller-manager/app/mizarcontrollers.go. The tests work.